From: Takashi Iwai <tiwai@suse.de>
Subject: Fix kABI breakage for of/device change
Patch-mainline: Never, kABI fix
References: bsc#1051510

The patch patches.drivers/of-restrict-DMA-configuration.patch added
a few new header file inclusions and this breaks kABI.
Wrap them with the usual magic ifdef.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/of/device.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/of/device.c
+++ b/drivers/of/device.c
@@ -9,9 +9,11 @@
 #include <linux/module.h>
 #include <linux/mod_devicetable.h>
 #include <linux/slab.h>
+#ifndef __GENKSYMS__
 #include <linux/pci.h>
 #include <linux/platform_device.h>
 #include <linux/amba/bus.h>
+#endif
 
 #include <asm/errno.h>
 #include "of_private.h"
